packages/authorization-updater/.env

```
@@ -0,0 +1,12 @@
+HOST=0.0.0.0
+PORT=3000
+LOG_LEVEL=info
+
+KAFKA_CLIENT_ID="authorization-updater-client"
+KAFKA_GROUP_ID="authorization-updater-group"
```

group id used for local tests should be different from deployed value (otherwise local execution would prevent deployed service to consume messages)
```suggestion
KAFKA_GROUP_ID="authorization-updater-group-local"
```